volestipy/volestipy/src/bindings.cpp

```diff
@@ -24,9 +26,11 @@ HPolytopeCPP::HPolytopeCPP(double *A_np, double *b_np, int n_hyperplanes, int n_
       }
    }
 
-   HP.init(n_variables, A, b);
-   CheBall = HP.ComputeInnerBall();
+   HP = Hpolytope(n_variables, A, b);
+   std::cout<<"A = "<<HP.get_mat()<<"\n b= "<<HP.get_vec()<<std::endl;
```

Comment:
remove printing or protect it by some debug flag

Reply:
Print statements were removed